pkg/policy/l4.go

```diff
@@ -463,7 +501,36 @@ func NewL4Policy(revision uint64) *L4Policy {
 		Ingress:  L4PolicyMap{},
 		Egress:   L4PolicyMap{},
 		Revision: revision,
+		users:    make(map[*EndpointPolicy]struct{}),
+	}
+}
+
+// insertUser adds a user to the L4Policy so that incremental
+// updates of the L4Policy may be fowarded to the users of it.
+func (l4 *L4Policy) insertUser(user *EndpointPolicy) {
+	l4.mutex.Lock()
+	if l4.users == nil {
+		log.Warningf("L4Filter: Skipping setting user on a stale policy")
+	} else {
+		l4.users[user] = struct{}{}
+	}
+	l4.mutex.Unlock()
+}
+
+// AccumulateMapChanges distriburtes the given changes to the registered users.
+func (l4 *L4Policy) AccumulateMapChanges(adds, deletes []identity.NumericIdentity,
+	port uint16, proto uint8, direction trafficdirection.TrafficDirection) {
+
+	if l4 == nil {
+		log.Warning("L4Filter: Skipping incremental updates due to nil policy pointer")
```

Comment:
Same here for the user inability of doing any action if it sees this message

Reply:
Changed to Debug